config/routes.rb

```diff
@@ -71,19 +71,27 @@
         get "record-template", on: :collection
       end
 
-      resource :nurse_consents, path: ":route/consent" do
-        get "assessing-gillick", to: "nurse_consents#assessing_gillick"
+      constraints -> { Flipper.enabled?(:new_consents) } do
+        resources :manage_consents,
+                  only: %i[show update],
+                  path: ":route/consents"
+      end
 
-        get "edit/gillick", to: "nurse_consents#edit_gillick"
-        put "update/gillick", to: "nurse_consents#update_gillick"
+      constraints -> { !Flipper.enabled?(:new_consents) } do
```

Comment:
Hrm. I love that we can feature flag this, but at which point do we want to test both paths to ensure the fallback is valid?

I guess we _could_ add this route to expose the new consents on `:route/new_consents`:

```
      constraints -> { !Flipper.enabled?(:new_consents) } do
        resources :manage_consents,
                  only: %i[show update],
                  path: ":route/new_consents"
      end
```

If we think it's worth it. Haven't quite yet decided myself yet.

Reply:
Yeh at some point we can revisit this 👍